test/test_pipeline.py

```
@@ -0,0 +1,169 @@
+"""
+Section 4 — Processing Pipeline Tests
+=======================================
+Autodiscovery: the four meaningful combinations of ``--correct`` × ``--summarize``
```

Ruff (0.15.15)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)